<dec f='include/c++/4.9/bits/exception_ptr.h' l='64' type='std::__exception_ptr::exception_ptr std::current_exception()'/>
<dec f='include/c++/4.9/bits/exception_ptr.h' l='86' type='std::__exception_ptr::exception_ptr std::current_exception()'/>
<use f='include/c++/4.9/bits/exception_ptr.h' l='179' u='c' c='_ZSt18make_exception_ptrT_'/>
<use f='include/c++/4.9/bits/nested_exception.h' l='60' u='c' c='_ZNSt16nested_exceptionC1Ev'/>
<doc f='include/c++/4.9/bits/exception_ptr.h' l='60'>/** Obtain an exception_ptr to the currently handled exception. If there
   *  is none, or the currently handled exception is foreign, return the null
   *  value.
   */</doc>
